frontend/components/Filter/FilterDialog.tsx

```diff
@@ -280,7 +280,10 @@ export default function FilterDialog({
                       </div>
 
                       <Datepicker
-                        value={dateRange}
+                        value={{
+                          startDate: dateRange && dateRange.startDate ? new Date(dateRange.startDate) : new Date(),
```

Comment:
this change was needed to fix up linting problems (which prevent me from comitting because of the precommit hook)